dracut/30ignition/module-setup.sh

```diff
@@ -54,7 +54,24 @@ install() {
 #       "$systemdsystemunitdir/coreos-static-network.service"
 }
 
+has_builtin_fw_cfg() {
+    # this is like check_kernel_config() but it specifically checks for `y` and
+    # also checks the OSTree-specific kernel location
+    for path in /boot/config-$kernel \
+                /usr/lib/modules/$kernel/config \
+                /usr/lib/ostree-boot/config-$kernel; do
+        if test -f $path; then
+            rc=0
+            grep -q CONFIG_FW_CFG_SYSFS=y $path || rc=$?
```

Comment:
would `grep -q CONFIG_FW_CFG_SYSFS=y $path || return $?` work?

Reply:
Hmm, I think so, but I really don't want to retest all those streams I just tested. :)

Reply:
That'd have to be:

```
grep -q ... || return $?
return 0
```

right?

Or I guess

```
(grep -q ... || return $?) && return 0
```

But that's way less clear than just typing it out (I'll mention that the approach here is one of the idiomatic ways to capture the return code in the presence of `set -e`; I wasn't sure if `set -e` was turned on in these scripts, so I went with that approach, which works regardless).